src/main/java/com/google/api/codegen/PageStreamingConfig.java

```diff
@@ -39,6 +40,11 @@
   @Nullable
   public static PageStreamingConfig createPageStreaming(
       DiagCollector diagCollector, PageStreamingConfigProto pageStreaming, Method method) {
+    // Since page_size is not required, no error should be raised if it's not present.
+    String pageSizeFieldName = pageStreaming.getRequest().getPageSizeField();
+    Field pageSizeField =
+        method.getInputType().getMessageType().lookupField(pageSizeFieldName);
```

Comment:
This should still cause an error.

- page_size is specified in config, page_size is not defined in proto ==> error
- page_size is not specified in config ==> pageSizeField is set to null